db/migrate/20230530101854_create_search_results.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :search_results do |t|
       t.string :keyword
       t.string :search_engine
+      t.text :html_code
```

Comment:
Normally we should never update already existing migrations, it's a better practice to create a new migration.

Imagine we have a lot of data in production, then we would need to drop first in order to migrate successfully.

But for this case, it's OK, just something to keep in mind 🙏 

@olivierobert @hoangmirs Please let me know if I'm wrong 🙇

Reply:
@Tuubz Good catch 👍 New migration file pushed 🚀 [f71716e](https://github.com/nimblehq/ic-rails-huey-toby/pull/30/commits/f71716ec795176418fe7e320578829d65d21e4c6)